deployment/run_server

```diff
@@ -6,6 +6,10 @@ if [[ $PLOTLY_CUSTOM_FONTS_ENABLED = true ]]; then
   fc-cache -v /usr/share/fonts/customer
 fi
 
+if [[ $PLOTLY_IMAGESERVER_IGNORE_CERT_ERRORS = true ]]; then
+  ORCA_IGNORECERTERRORS_ARG="--ignore-certificate-errors"
```

Comment:
For reference, flag [`--ignore-certificate-errors` is documented here in Electron 1.8](https://github.com/electron/electron/blob/1-8-x/docs/api/chrome-command-line-switches.md#--ignore-certificate-errors) and is available in newer versions of Electron as well!